core/src/test/java/greencity/controller/SuperAdminControllerTest.java

```diff
@@ -255,7 +255,7 @@ void updateCourierTest() {
         ObjectMapper objectMapper = new ObjectMapper();
         String json = objectMapper.writeValueAsString(dto);
 
-        mockMvc.perform(put(ubsLink + "/update-courier")
+        mockMvc.perform(put(ubsLink + "/editCourier")
```

Comment:
I see we have a bunch of possible response codes for `/editCourier` endpoint
```
@ApiResponses(value = {
        @ApiResponse(code = 200, message = HttpStatuses.OK, response = CourierDto.class),
        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND),
        @ApiResponse(code = 422, message = HttpStatuses.UNPROCESSABLE_ENTITY)
    })
```
Could you please add tests for all these cases?